frontend/app/src/App.tsx

```diff
@@ -852,6 +855,7 @@ export class App extends PureComponent<Props, State> {
       totalLoadTime: Math.round(
         (performance.now() - this.state.latestRunTime) * 1000
       ),
+      browserInfo,
```

Comment:
is it cool for this to be a nested object?

Reply:
I think it is fine. But maybe also double check with the data team.

Reply:
fine for segment, not fine for fivetran so going to flatten now